AutocompleteClient/FW/Logic/Request/CIOTrackConversionData.swift

```diff
@@ -19,20 +19,24 @@ struct CIOTrackConversionData: CIORequestData {
     var sectionName: String?
     let revenue: Double?
     let conversionType: String?
+    let displayName: String?
+    let isCustomType: Bool?
```

Comment:
Do we need both fields to be added to `RequestBuilder+QueryItems.swift`?

https://github.com/Constructor-io/constructorio-client-swift/blob/6ba67244145ec1022c29f9d252d6e152249be94d/AutocompleteClient/FW/Logic/Request/Builder/RequestBuilder%2BQueryItems.swift#L1